### ISSUE TITLE
Cannot read property 'stdin' of null

### ISSUE THREAD

**Issue by cibulka** [w0rm/gulp-svgfallback#7]
I've started to recieve this error, not sure what changed in my setup to cause this. :(

```
/Users/Cibulka/Documents/Web/itvar.dev/node_modules/cibulka-gulp-tasks/node_modules/gulp-svgfallback/node_modules/phridge/lib/Phantom.js:234
    this.childProcess.stdin.write(JSON.stringify(message) + os.EOL, "utf8");
                     ^
TypeError: Cannot read property 'stdin' of null
    at Phantom._write (/Users/Cibulka/Documents/Web/itvar.dev/node_modules/cibulka-gulp-tasks/node_modules/gulp-svgfallback/node_modules/phridge/lib/Phantom.js:234:22)
    at timer._onTimeout (timers.js:232:16)
    at Timer.listOnTimeout (timers.js:119:15)
```

My NPM/Gulp setup is a little funky, so if you can't tell the issue at first glance, I'm definitely able to provide more context. Thanks a lot!

**Comment by w0rm** [comment]
Hi! Thanks for the issue, I will look into it tonight. Have you tried this in a clean environment?

**Comment by cibulka** [comment]
Not yet, since the PNG fallbacks are not essential for the project (which is, as always, on a tight deadline :)). 

So far I've only turned off the part of the plugin containing `gulp-svgfallback` to confirm that the issue lies there. I will be able to do some more debugging in the beginning of the next week, if that's okay.

Thank you for quick answer!

**Comment by cibulka** [comment]
Just a quick note before debugging Monday: I use `gulp-svgfallback` inside loop and with `merge-stream` plugin like this:

```
module.exports = function(options, config, $) {

    var merged = $.mergeStream();

    merged.add(
        $.gulp.src(config.dest.ico + '*min.svg')
            .pipe(svgSprite(options, $))
            .pipe($.gulp.dest(config.dest.sprites))
    );

    if (options.useSvgFallback) {
        merged.add(
            $.gulp.src(config.dest.ico + '*min.svg')
                .pipe($.svgfallback())
                .pipe($.gulp.dest(config.dest.sprites))     
        );
    } else {
        console.log("Skipping SVG Fallback.");
    }

    return merged;

};
```

This make it seem to be related to https://github.com/giakki/uncss/issues/136.

**Comment by w0rm** [comment]
Looks like the error is coming from `phridge`. Can you tell what is `svgSprite`, can it be that it also uses `phridge`?

**Comment by w0rm** [comment]
Never mind my question, found this in your profile that only `svgfallback` uses `phridge`.
I don't really know what may cause this. A minimal test case that I can run locally would be helpful to investigate this further e.g. a gulpfile with just one task and source svgs.

**Comment by jhnns** [comment]
@cibulka please re-install `gulp-svgfallback` and verify that `phridge@1.2.2` is installed. This issue should have been fixed with `1.2.2`.

**Comment by cibulka** [comment]
Yes, that fixed it! Sorry for my late and yay for your quick responses!
